src/css/profile/mobile/themes/dark.variables.less

```diff
@@ -41,8 +41,9 @@
     @button-background-contained:  @color-white;
     @text-input-invalid-color: #ff6666;
 
-    @dropdown-menu-options-border: #525252;
+    @dropdown-menu-options-border: 3 * @px_base solid #525252;
```

Comment:
I wondering about this and I not sure but I don't found possible issues referring to complex variable in this way

Reply:
I can split this to two variables:
 - @dropdown-menu-options-border-width
 - @dropdown-menu-options-border-color

Reply:
This option surprised me. I didn't think about this way to define styles before. You decide.

Reply:
I leave it as it is to avoid more variables in the code.